addons/spreadsheet/static/tests/widgets/spreadsheet_binary_field.test.js

```diff
@@ -0,0 +1,53 @@
+import { expect, test } from "@odoo/hoot";
+import { defineModels, fields, models, mountView, onRpc } from "@web/../tests/web_test_helpers";
+import { defineMailModels } from "@mail/../tests/mail_test_helpers";
+import { click } from "@odoo/hoot-dom";
+import { animationFrame } from "@odoo/hoot-mock";
+
+class SpreadsheetDashboard extends models.Model {
+    _name = "spreadsheet.dashboard";
```

Comment:
This is actually not coupled to dashboards, but could be applied to any spreadsheet model. I'd rename this to make it clear the widget is generic
```suggestion
class TestSpreadsheet extends models.Model {
    _name = "test.spreadsheet";
```